packages/iov-core/src/integrationtests/atomicswapbnsbns.spec.ts

```diff
@@ -27,7 +27,7 @@ import { Ed25519HdWallet, HdPaths, UserProfile } from "@iov/keycontrol";
 import { MultiChainSigner } from "../multichainsigner";
```

Comment:
Maybe this file should be in the BNS package rather than Core? We have an analogous integration test in the Ethereum package.

Reply:
Very good idea. It just tried that. Unfortunately, the test is based on the MultiChainSigner, which means a lot of refactoring is required to use lower-level equivalents (explicit signing and posting). So this is out of scope for this PR but definitely worth doing.  https://github.com/iov-one/iov-core/issues/1035